src/current/v23.1/connection-pooling.md

```diff
@@ -49,14 +65,46 @@ If you have a large number of services connecting to the same cluster, make sure
+
+Do not set the connection pool timeout values to be too short, as it may cause the connection pool software to close and reopen connections frequently, causing increased latency. [Monitor the `sql.new_conns` metric](#monitor-new-connections) to make sure the timeout values are set correctly.
+
 {% include {{page.version.version}}/sql/server-side-connection-limit.md %} This may be useful in addition to your connection pool settings.
 
-## Validating connections in a pool
+</section>
+
+### Size connection pools in multi-region clusters
+
+<section class="filter-content" markdown="1" data-scope="serverless">
+
+Similar to single-region {{ site.data.products.serverless }} clusters, the purpose of connection pools in multi-region {{ site.data.products.serverless }} clusters is to reduce connection latency. Add a connection pool for each region, and size the idle pool for your application and workload.
+
+The cluster will automatically scale up to meet demand, so there's no need to set a maximum connection pool size.
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="dedicated selfhosted">
+
+For multi-region clusters, create a connection pool per region, and size the maximum connection pool for each region in your cluster using the same formula as a single-region cluster.
+
+For example, if you have 3 regions in your cluster, and each region has 12 vCPUs, create a connection pool for each region, with each connection pool having a maximum pool size of 48 (12 [processor cores] * 4).
+
+</section>
+
+## Validate connections in a pool
 
 After a connection pool initializes connections to CockroachDB clusters, those connections can occasionally break. This could be due to changes in the cluster topography, or rolling upgrades and restarts, or network disruptions. {{ site.data.products.db }} clusters periodically are restarted for patch version updates, for example, so previously established connections would be invalid after the restart.
 
 Validating connections is typically handled automatically by the connection pool. For example, in HikariCP the connection is validated whenever you request a connection from the pool, and the `keepaliveTime` property allows you to configure an interval to periodically check if the connections in the pool are valid. Whatever connection pool you use, make sure connection validation is enabled when running your application.
 
+<section class="filter-content" markdown="1" data-scope="dedicated selfhosted">
+
+## Monitor new connections
+
+The [`sql.new_conns` metric](metrics.html#available-metrics) exposes the number of new SQL connections per second. A properly configured connection pool will show this value to be in the low single digits. A misconfigured connection pool will result in much higher values. You can expose and monitor this metric in the [DB Console](ui-custom-chart-debug-page.html).
```

Comment:
Can we add some treatment here to suggest connection pools jitter their connection, please?

> Periodic spikes in new connections, also known as thundering herds or connection storms, should be avoided.  Most connection pools offer the ability to stagger their connection age through the use of a connection age "jitter."  Using a connection jitter of 10% of the connection age should be sufficient to smooth out the rate of new connections to a cluster (e.g. max connection age of 300s and a connection age jitter of 30s).